src/components/FlagButton/FlagButton.tsx

```diff
@@ -8,6 +8,7 @@ import { Styled } from './FlagButton.styled'
 
 export type FlagButtonProps = IconButtonProps & {
   isoCode: MuiTelInputCountry | null
+  countryName: string | undefined
```

Comment:
but I don't get it, we can have conflict like "isoCode" to "FR" and "countryName" to "Spain"..

Reply:
As I mentioned before, we should also avoid this conflict inside the `Flag` componet since it can also happen there. So, what you think if I pass the `langOfCountryName` for this component?